build/build.sh

```diff
@@ -25,10 +25,10 @@ then
 		sbt tut 'release with-defaults'
 	elif [[ $TRAVIS_BRANCH == "master" ]]
 	then
-		sbt clean coverage test tut coverageReport coverageAggregate checkUnformattedFiles
+		sbt clean coverage +test tut coverageReport coverageAggregate checkUnformattedFiles
 		sbt coverageOff publish
 	else
-		sbt clean coverage test tut coverageReport coverageAggregate checkUnformattedFiles
+		sbt clean coverage +test tut coverageReport coverageAggregate checkUnformattedFiles
```

Comment:
seems like this didn't work also, I don't see any `2.12` appearances in travis logs